src/vanclj/core.clj

```diff
@@ -65,7 +65,7 @@
 (defn member-section [members]
   [:div.tile.is-ancestor
    [:div.tile.is-vertical
-    (for [members (partition-all 4 (take 8 members))]
+    (for [members (partition-all 4 members)]
```

Comment:
So the rationale around this was: it gave us the ability to keep things a bit more up-to-date. Some members are here but I do not feel removing them even if they do not participate anymore.

For that reason, I would keep the `take 8` :smile: 

We might want to consider having an alumni section but haven't had time for that. 

Please add your avatar in place of Tavis Tudd (or I can do it :smile:).

Oh...if the group grows we will increase the number.

Reply:
How about `(take 8 (shuffle members))`?

Reply:
Oh that's a good idea

Reply:
The only thing I am realizing now is that this is not dynamic but static so it will change only every time I recompile had publish.